modules/custom/az_flexible_page/config/install/views.view.az_page_by_category.yml

```diff
@@ -274,16 +273,18 @@ display:
           col_md: col-md-4
           col_lg: col-lg-4
           col_xl: col-xl-4
-      defaults:
-        style: false
-        row: false
-        style_options: false
       row:
         type: 'entity:node'
         options:
           relationship: none
           view_mode: az_card
+      defaults:
+        style: false
+        style_options: false
+        row: false
+      display_description: ''
       style_options: null
+      display_extenders: {  }
```

Comment:
FYI in case you were wondering what display_extenders are https://studio.gd/node/17

Reply:
Woah, thank you!